src/Hintman/Core/Hint.hs

```diff
@@ -30,24 +27,11 @@ parseHintType :: Text -> Maybe HintType
 parseHintType = inverseMap showHintType
 {-# INLINE parseHintType #-}
 
-
-
 data Line = Line
     { lineNumber :: !Int
     , lineBody   :: !Text
     } deriving (Eq, Show)
 
-data Suggestion = Suggestion
-    { suggestionFile   :: FilePath
-    , suggestionLine   :: Line
-    , suggestionChange :: LineChange
-    } deriving (Eq, Show)
-
-data LineChange = LineChange
-    { lineChangeNew     :: Text
-    , lineChangeComment :: Maybe Text
-    , lineChangeType    :: HintType
-    } deriving (Eq, Show)
-
```

Comment:
Thanks for the cleanup! I see that this is not needed anymore and we can reuse existing types.